selendroid-common/src/main/java/io/selendroid/common/SelendroidCapabilities.java

```diff
@@ -395,4 +395,16 @@ private String getDefaultVersion(Set<String> keys, String appName) {
     return listOfApps.size() > 0 ? listOfApps.last() : null;
   }
 
+  // throws exception if user didn't pass the capability as a boolean
+  private Boolean getBooleanCapability(String key) {
+    Object o = getRawCapabilities().get(key);
+    if (o == null) {
+      return null;
+    } else if (o instanceof Boolean) {
+      return (Boolean) o;
+    } else {
+      throw new ClassCastException("DesiredCapability " + key
```

Comment:
this seems consistent with the exception that was previously thrown but has a more useful error message.  I think if you could add `o.toString()` in the message, that might be event more helpful:

`throw new ClassCastException(String.format("DesiredCapability %s's value should be boolean: found value %s of type %s", key, o.toString(), o.getClass().getName()))`